### PR TITLE
Added simulate mode in Hive JDBC Connector to simulate query execution

### PR DESCRIPTION
**Pull request by abti**
Added simulate mode in Hive JDBC Connector to simulate query execution

### CODE REVIEW

**Comment by abti** [comment]
@pcadabam Please review

File: gobblin-compaction/src/main/java/gobblin/hive/util/HiveJdbcConnector.java [hunk 1]
```diff
@@ -86,7 +95,10 @@ public static HiveJdbcConnector newEmbeddedConnector(int hiveServerVersion) thro
    * @return
    */
   public static HiveJdbcConnector newConnectorWithProps(Properties compactRunProps) throws SQLException {
-    HiveJdbcConnector hiveJdbcConnector = new HiveJdbcConnector();
+
+    boolean isSimulate = "true".equalsIgnoreCase(compactRunProps.getProperty(HIVE_EXECUTION_SIMULATE, "false"));
```
Comment (pcadabam) [inline gobblin-compaction/src/main/java/gobblin/hive/util/HiveJdbcConnector.java]: Use ```boolean isSimulate = Boolean.valueOf(compactRunProps.getProperty(HIVE_EXECUTION_SIMULATE))```
https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#valueOf(java.lang.String)

**Comment by pcadabam** [comment]
LGTM apart from that one comment

File: gobblin-compaction/src/main/java/gobblin/hive/util/HiveJdbcConnector.java (hunk 1 above)
Comment (abti) [inline gobblin-compaction/src/main/java/gobblin/hive/util/HiveJdbcConnector.java]: Changed

**Comment by abti** [comment]
Thanks for the review

**Pull request merged**